book.toml

```diff
@@ -46,7 +46,7 @@ urlcolor = "red"
 
 [output.html]
 curly-quotes = true
-additional-js = ["theme/speaker-notes.js", "theme/redbox.js"]
+additional-js = ["theme/speaker-notes.js", "theme/additional.js"]
```

Comment:
Ah, I thought you meant adding a _new_ JS file for this functionality, rather than moving the existing red-box functionality too. Maybe keep `theme/redbox.js` and move this new functionality into `theme/save-playgrounds.js`?